src/components/transport_manager/src/bluetooth/bluetooth_device_scanner.cc

```diff
@@ -213,39 +213,7 @@ void BluetoothDeviceScanner::DoInquiry() {
       paired_devices_, device_handle, &paired_devices_with_sdl_);
   UpdateTotalDeviceList();
 
-  LOG4CXX_INFO(logger_, "Starting hci_inquiry on device " << device_id);
-  const uint8_t inquiry_time = 8u;  // Time unit is 1.28 seconds
-  const size_t max_devices = 256u;
-  inquiry_info* inquiry_info_list = new inquiry_info[max_devices];
-
-  const int number_of_devices = hci_inquiry(device_id,
-                                            inquiry_time,
-                                            max_devices,
-                                            0,
-                                            &inquiry_info_list,
-                                            IREQ_CACHE_FLUSH);
-
-  if (number_of_devices >= 0) {
-    LOG4CXX_INFO(logger_,
-                 "hci_inquiry: found " << number_of_devices << " devices");
-    std::vector<bdaddr_t> found_devices(number_of_devices);
-    for (int i = 0; i < number_of_devices; ++i) {
-      found_devices[i] = inquiry_info_list[i].bdaddr;
-    }
-    found_devices_with_sdl_.clear();
-    CheckSDLServiceOnDevices(
-        found_devices, device_handle, &found_devices_with_sdl_);
-  }
-  UpdateTotalDeviceList();
-  controller_->FindNewApplicationsRequest();
```

Comment:
Core is able to detect sdl apps on my connected bluetooth device so I don't think this call is required.

Also, since the blocking hci_inquiry call is removed and the `auto_repeat_pause_sec` is set to 0, the `TimedWaitForDeviceScanRequest` doesn't cause any delay in the thread causing core to repeatedly send `UpdateAppList` requests to the HMI

![Screenshot from 2020-06-18 10-50-50](https://user-images.githubusercontent.com/10703941/85038131-399a3c80-b154-11ea-8a69-9b6ae5809561.png)

Reply:
This is resolved now